adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/AdyenSummaryCheckoutStepController.java

```diff
@@ -209,11 +238,15 @@ public String authorise3DSecurePayment(final Model model,
             return redirectToOrderConfirmationPage(orderData);
         } catch (AdyenNonAuthorizedPaymentException e) {
             PaymentResult paymentResult = e.getPaymentResult();
+            PaymentsResponse paymentsResponse = e.getPaymentsResponse();
```

Comment:
At a given time, only paymentResult or paymentsResponse will be present. if paymentResult is null then only paymentResponse should be retrieved.